tests/test_processors.py

```diff
@@ -456,6 +456,17 @@ def test_obtains_exc_info_on_True(self):
         else:
             pytest.fail("Exception not raised.")
 
+    def test_obtains_exc_info_on_number(self):
```

Comment:
Yep pytest is a treasure chest; spending some time in its docs pays off big time. :)